docs/src/01adr-cli.md

```diff
@@ -0,0 +1,88 @@
+# ADR-01: Atomkraft CLI
+
+| authors           | revision | revision date  |
+| ----------------- | --------:| --------------:|
+| Andrey Kuprianov  |        1 | July 11, 2022  |
+
+This ADR describes the command-line interface of Atomkraft, and how it is supposed to be used by the users as a standalone tool. It *does not* describe other possible usage scenarios, e.g. integration with testing frameworks (such as Pytest). We structure the description around the tool use cases.
+
+Below we specify only the outcomes for successful command execution. Upon unsuccessful command execution, the error should be reported to the user, and no remnants (e.g. zombie processes, or additional files beyond requested) should remain.
+
+## Launch a testnet
+
+A user wants to set up a Cosmos-SDK testnet, using a specific binary of their choice, so that the testnet can later be used either for exploration or testing.  
+
+Upon successful command execution, the testnet should be in operational step, and keep running; the testnet mnemonic is returned to the user. The proposed command format:
+
+```
+atomkraft testnet <binary> <genesis-config> <node-config>
```

Comment:
```suggestion
atomkraft testnet <binary> [<genesis-config>] [<node-config>]
```
I suggest turning configs into optional arguments: if not specified, some healthy defaults are used. In line with the previous comment, if we are using only a set of supported binaries, we can have meaningful defaults specified for them